components/brave_wallet_ui/constants/types.ts

```diff
@@ -685,3 +685,15 @@ export enum WalletOnboardingSteps {
   OnboardingImportMetaMask = 3,
   OnboardingImportBraveLegacy = 4
 }
+
+export enum WalletRoutes {
+  Unlock = '/crypto/unlock',
+  Onboarding = '/crypto/onboarding',
+  Restore = '/crypto/restore-wallet',
+  Portfolio = '/crypto/portfolio',
+  PortfolioSub = '/crypto/portfolio/:id?',
+  Accounts = '/crypto/accounts',
+  AccountsSub = '/crypto/accounts/:id?',
+  Backup = '/crypto/backup-wallet',
+  CryptoPage = '/crypto/:category/:id?'
+}
```

Comment:
It seems all routes have the `/crypto/` prefix, which makes it superfluous. Is there any case where the prefix is something else? If not, why not drop it altogether and have routes like `/unlock`, `/onboarding`, etc, since there's already the global `/wallet` prefix?

Reply:
When we introduce `Rewards` and `Cards` there will be prefix's for those as well for example
`brave://wallet/rewards` and `brave://wallet/cards`, etc..